Estore.UITests/StepDefinitions/NavigationSteps.cs

```diff
@@ -19,22 +22,69 @@ public NavigationSteps(DataContext context, IConfiguration configuration)
             _configuration = configuration;
         }
 
-        [Given(@"open users page")]
+        [Given(@"Open users page")]
         public void GivenOpenUsersPage()
         {
-            _context.Driver.Navigate().GoToUrl($"{_baseUrl}{_configuration["Pages:users"]}");
-            _context.UserPage = new UsersPage(_context.Driver);
-            _context.CurrentPage = _context.UserPage;
-            _context.UserPage.LoadUserTable();
+            _context.Driver.Navigate().GoToUrl($"{_baseUrl}{_configuration["Pages:users"]}");            
         }
 
-        [Given(@"open main page")]
+        [Given(@"Open main page")]
         public void GivenOpenMainPage()
         {
-            _context.Driver.Navigate().GoToUrl($"{_baseUrl}{_configuration["Pages:main"]}");
+            _context.Driver.Navigate().GoToUrl($"{_baseUrl}{_configuration["Pages:main"]}");            
+        }
+
+        [Given(@"Open login page")]
+        public void GivenOpenLoginPage()
+        {
+            _context.Driver.Navigate().GoToUrl($"{_baseUrl}{_configuration["Pages:login"]}");                                    
+        }
+
+        [Given(@"Login page is open")]
+        [When(@"Login page is open")]
+        [Then(@"Login page should be open")]
+        public void LoginPageIsOpen()
+        {
+            _context.LoginPage = new LoginPage(_context.Driver);
+        }
+
+        [Given(@"Main page is open")]
+        [When(@"Main page is open")]
+        [Then(@"Main page should be open")]
```

Comment:
Is it possible to make the common name for given/when/then? If yes, it is possible to use [StepDefinition(expression)]

Reply:
fixed